### PR TITLE
Readme fixes curl mtls

### PR DESCRIPTION
**Pull request by adarshan-intel**
This fixes - https://jira.devtools.intel.com/browse/SDOPT-12786

### CODE REVIEW

File: component-samples/demo/README.MD [hunk 1]
```diff
@@ -299,7 +299,7 @@ Finally, the mTLS enabled cURL request will be:
     --cacert scripts/secrets/ca-cert.pem --cert scripts/api-user.pem \
      --location --request POST 'http://host.docker.internal:8039/api/v1/rvinfo' \
     --header 'Content-Type: text/plain' \
-    --data-raw '[[[5,"{rv-dns}"],[3,8040],[12,1],[2,"{rv-ip}"],[4,8040]]]'
+    --data-raw '[[[5,"\{rv-dns}"],[3,8040],[12,1],[2,"\{rv-ip}"],[4,8040]]]'
```
Comment (DukeDavis12) [inline component-samples/demo/README.MD:302]: image
Why do we need this change ?

Comment (adarshan-intel) [reply component-samples/demo/README.MD:302]: This change is made according to this JIRA issue - https://jira.devtools.intel.com/browse/SDOPT-12786

Comment (Sai-Anudeep47) [reply component-samples/demo/README.MD:302]: Backslash is not required and adds more confusion.

Comment (adarshan-intel) [reply component-samples/demo/README.MD:302]: @Sai-Anudeep47 Alright should I close this PR ?

Comment (Sai-Anudeep47) [reply component-samples/demo/README.MD:302]: Yes, we can close this

**Pull request closed without merging**